src/views/blocks/tickets/content-inactive.php

```diff
@@ -15,8 +15,7 @@
  *
  */
 
-$message = $this->get( 'is_sale_past' ) ? __( 'Tickets are no longer available', 'event-tickets' ) : __( 'Tickets are not yet available', 'event-tickets' );
-?>
+$message = $this->get( 'is_sale_past' ) ? sprintf( __( '%s are no longer available', 'event-tickets' ), tribe_get_ticket_label_plural( 'event-tickets' ) ) : sprintf( __( '%s are not yet available', 'event-tickets' ), tribe_get_ticket_label_plural( 'event-tickets' ) );
```

Comment:
When we use a placeholder in translations we need to include the php comment describing what the placeholder is, like this: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#multiline-function-calls